datadog_checks_dev/datadog_checks/dev/tooling/commands/release/trello/tester_selector/tester_selector_team.py

```diff
@@ -38,7 +38,7 @@ def get_next_tester(self, author: str, pr_num: int) -> Optional[str]:
         tester = self.__select_testers(lambda t: t in exclude_testers)
         if tester is None:
             # find a tester who is not the author
-            tester = self.__select_testers(lambda t: t != author)
+            tester = self.__select_testers(lambda t: t == author)
```

Comment:
```suggestion
            tester = self.__select_testers(user_excluded_fct=lambda t: t == author)
```
Can we do sth like this to make it explicit that is an exclusion?

Reply:
We can yes, but maybe we can do it in both callsites ?